src/main/java/io/anserini/qa/passage/IdfPassageScorer.java

```diff
@@ -101,14 +118,19 @@ public void score(String query, Map<String, Float> sentences) throws Exception {
           TermQuery q = (TermQuery) qp.parse(term);
           Term t = q.getTerm();
           double termIDF = similarity.idf(reader.docFreq(t), reader.numDocs());
-          termIdfMap.put(term, String.valueOf(termIDF));
 
           if (questionTerms.contains(t.toString()) && !seenTerms.contains(t.toString())) {
```

Comment:
The intuition is that the answer that have rare question terms are important. Earlier, while working on the E2E demo, we realized that considering the term frequency of the question terms lowered the quality of the output. This, perhaps, is due to the repetition of common question terms in the answers.